worlds/stardew_valley/data/items.csv

```diff
@@ -941,15 +943,20 @@ id,name,classification,groups,mod_name
 10702,Resource Pack: 3 Travel Core,filler,RESOURCE_PACK,Magic
 10703,Preservation Chamber,filler,"RESOURCE_PACK,RESOURCE_PACK_USEFUL",Archaeology
 10704,Hardwood Preservation Chamber,filler,"RESOURCE_PACK,RESOURCE_PACK_USEFUL",Archaeology
-10705,Resource Pack: 3 Water Shifter,filler,RESOURCE_PACK,Archaeology
+10705,Resource Pack: 3 Water Sifter,filler,RESOURCE_PACK,Archaeology

```

Comment:
I generally agree with using the display names in many places, it makes sense to me that the AP items and locations are display names, not internal names.

But for the Water Shifter specifically, I think it should be "Water Shifter". "Sifter" is clearly a typo, not an intended player-facing name, and hopefully it can be fixed. While  can't find much reference in real life to that machine, I took a quick look at the archaeology code, and it's using "Shifter" pretty much **everywhere**, so I'm almost certain it's the intended name.